src/easier68k/core/opcodes/dc.py

```diff
@@ -0,0 +1,392 @@
+from ...core.opcodes.opcode import Opcode
+from ...core.util import opcode_util
+from ...simulator.m68k import M68K
+from ..util.parsing import parse_literal
+import math
+
+
+def command_matches(command: str) -> bool:
+    """
+    Checks whether a command string is an instance of this command type
+    :param command: The command string to check (e.g. 'MOVE.B', 'LEA', etc.)
+    :return: Whether the string is an instance of this command type
+    """
+    return opcode_util.command_matches(command, 'DC')
+
+
+class_name = 'DC'
+
+
+class DC(Opcode):
+    allowed_sizes = 'BWL'
+
+    def __init__(self, values: list, size='W'):
+        assert size.upper() in DC.allowed_sizes
+        self.size = size
+
+        assert len(values) > 0
+        self.values = values
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        """
+        Parses a command string into an instance of the opcode class
+
+        >>> test0 = DC.from_str("DC.B", "$0A, $0B")[0]
+        >>> test0.size
+        'B'
+        >>> test0.values
+        ['$0A', '$0B']
+
+        >>> test1 = DC.from_str("DC.B", "\\'Hai!\\'")[0]
+        >>> test1.size
+        'B'
+        >>> test1.values
+        ['$48', '$61', '$69', '$21']
```

Comment:
Good question, I think I can convert it to numbers later on; I think it happened because I was doing some weird stuff to make sure the zero-padding got saved right, but looking at how it's set up right now I can probably make the final step convert to numbers instead of strings.